packages/connect-plugin-stellar/package.json

```
@@ -0,0 +1,36 @@
+{
+    "name": "@trezor/connect-plugin-stellar",
+    "version": "9.0.0",
+    "author": "Trezor <info@trezor.io>",
+    "homepage": "https://github.com/trezor/trezor-suite/packages/connect-plugin-stellar",
```

```suggestion
    "homepage": "https://github.com/trezor/trezor-suite/tree/develop/packages/connect-plugin-stellar",
```
i've [recently updated](https://github.com/trezor/trezor-suite/commit/deb62e4b0399e57915c2cac34b7dde5ddc40f5b6) homepage links to valid (this one is 404)

---

changed